src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -392,9 +393,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </ItemGroup>
 
   <ItemGroup Condition="'$(ProduceReferenceAssembly)' == 'true'">
-    <IntermediateRefAssembly Include="$(IntermediateOutputPath)ref\$(TargetName)$(TargetExt)" Condition="'@(IntermediateRefAssembly)' == ''" />
+    <IntermediateRefAssembly Include="$(IntermediateOutputPath)refint\$(TargetName)$(TargetExt)" Condition="'@(IntermediateRefAssembly)' == ''" />
     <CreateDirectory Include="@(IntermediateRefAssembly->'%(RootDir)%(Directory)')" />
-    <CreateDirectory Include="$(OutDir)ref" />
+    <CreateDirectory Include="$(IntermediateOutputPath)ref" />
```

Comment:
I feel like `$(IntermediateOutputPath)refint` should be added to this item. Does adding it to `IntermediateRefAssembly` ensure the path is getting created elsewhere?

Reply:
That's what the prior entry does:

```
@(IntermediateRefAssembly->'%(RootDir)%(Directory)')
```

Reply:
Shouldn't this and the above change also be under the opt-in property?